tests/common.bash

```diff
@@ -80,10 +80,29 @@ function handle_error() {
 }
 trap 'handle_error $LINENO' ERR
 
+# A wrapper function for kubectl with retry logic
```

Comment:
I feel so happy when I see functions documented \o/

Reply:
Could you more elaborate on this? I've already explained about the function over 3 lines of comment. What does `functions documented` mean?